src/test/java/com/esri/core/geometry/TestCut.java

```diff
@@ -51,7 +51,7 @@ public static void testCut4326() {
 
 	}
 
-	public static void testConsiderTouch1(SpatialReference spatialReference) {
+	private static void testConsiderTouch1(SpatialReference spatialReference) {
 		OperatorFactoryLocal engine = OperatorFactoryLocal.getInstance();
```

Comment:
Should all these tests be private?

Reply:
Those are not real tests but a methods that real tests call.